blast_config/tx_struct_tng.c

```diff
@@ -1027,7 +788,87 @@ channel_t channel_list[] =
+    { "int_ifyawmag2",      0.1 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifrollmag2",      0.1 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifrollxsc0",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifyawxsc0",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifelxsc0",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifrollxsc1",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifyawxsc1",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+    { "int_ifelxsc1",      100.0 / 32768.0,    0.0, TYPE_INT16, RATE_100HZ, U_NONE, 0 },
+
+    // pointing solution generating channels
+    { "thresh_cmd_atrim",         10.0 / 65536.0,   0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "time_cmd_atrim",           SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "rate_cmd_atrim",           30.0 / 65536.0,   0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "rate_atrim",           30.0 / 65536.0,   0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "thresh_atrim",           10.0 / 65536.0,   0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "fresh_trim",           SCALE(CONVERT_UNITY), TYPE_UINT8, RATE_5HZ, U_NONE, 0 },
+    { "new_az",               720.0 / 65536.0,  0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "new_el",               720.0 / 65536.0,  0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "weight_az",               360.0 / 65536.0,  0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "weight_el",               360.0 / 65536.0,  0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+
+    // null az and el trims from commands
+    { "trim_null",            I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "trim_el_null",            I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+
+    // slew veto, unsure of meaning
+    { "slew_veto",            SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "sveto_len",            SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+
+    // Scan parameters
+    // so many RA/DEC since we use 4 for quad boxes
+    { "ra_1_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_1_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_2_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_2_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_3_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_3_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_4_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_4_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
 
+    // we should comment this better...
```

Comment:
If it's possible to do so now, do it now

Reply:
Added some comments based on the command_struct.h declaration of these data parameters